workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer/src/api/azure.ts

```diff
@@ -14,10 +14,8 @@
  * limitations under the License.
  */
 import { createApiRef } from '@backstage/core-plugin-api';
-import {
-  AzureDevOpsApi,
-  getAnnotationValuesFromEntity,
-} from '@backstage-community/plugin-azure-devops';
+import { AzureDevOpsApi } from '@backstage-community/plugin-azure-devops';
```

Comment:
Not for this PR but would it make sense for `AzureDevOpsApi` to be moved to common as well? Seems heavy handed to bring in the entire frontend plugin to get access to this? 🤔